src/components/Atoms/SelectField/SelectField.js

```diff
@@ -83,13 +83,16 @@ const Select = ({
   ...rest
 }) => {
   const error = errorMsg && errorMsg.length > 0;
+
   return (
     <Label>
       <LabelText hideLabel={hideLabel} weight="bold">
         {label}
       </LabelText>
-      <StyledInput {...rest} error={error ? 1 : 0}>
-        <option disabled>{description}</option>
+      <StyledSelect {...rest} error={error ? 1 : 0}>
```

Comment:
can't error just be `error={error}` ?

Reply:
Yes I would have thought so but this isn't in my changes, I was just renaming the component as StyledInput confused me a little bit. Not entirely sure what the error attribute is, although looks like it might be passing a value to styled components?